documentation/book/assembly-healthchecks.adoc

```diff
@@ -12,8 +12,8 @@
 
 = Healthchecks
 
-Healthchecks are periodical tests which verify that the application's health.
-When the Healthcheck fails, {ProductPlatformName} can assume that the application is not healthy and attempt to fix it.
+Healthchecks are periodical tests which verify the health of an application.
+When a Healthcheck probe fails, {ProductPlatformName} assumes that the application is not healthy and attempts to fix it.
```

Comment:
Sounds good. I also edited this sentence but on a different branch! Please use your version when resolving the merge conflict.

Reply:
Will do.